pkg/api/v1alpha1/fluxconfig_test.go

```diff
@@ -1,13 +1,24 @@
 package v1alpha1
 
 import (
+	"os"
 	"reflect"
 	"testing"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	validPassword           = "testPassword"
+	eksaGitPasswordTokenEnv = "EKSA_GIT_PASSWORD"
+)
+
+func SaveContext() {
```

Comment:
why is this method public?
Also we should cleanup the env var after the test. You can use t.Cleanup
There are exampled like this somewhere, maybe vsphere

Reply:
we're now on 1.17 so you can use `t.Setenv` to set it for the duration of the test only too